README.md

```diff
@@ -45,23 +51,25 @@ may also be helpful.
 
 ## Quick usage
 
+<!---
```

Comment:
not sure to understand the question: you propose to have
```
<!---
## Quick usage
```
rather than
```
## Quick usage
<!---
```
right?